source/globalCommands.py

```diff
@@ -4443,6 +4444,47 @@ def script_cycleParagraphStyle(self, gesture: "inputCore.InputGesture") -> None:
 		config.conf["documentNavigation"]["paragraphStyle"] = newFlag.name
 		ui.message(newFlag.displayString)
 
+	@staticmethod
+	def _cycleOutputAudioDevices(forward: bool = True):
+		"""Cycle through available output devices.
+		@param forward: boolean flag if false cycles in backward direction.

```

Comment:
please update all the docstrings to use sphinx format

https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md#docstrings